tests/Unit/Models/RatingTest.php

```diff
@@ -40,4 +40,92 @@ public function can_belong_to_author()
         $rating = Rating::factory(1)->create()[0];
         $this->assertCount(1, $rating->author()->get());
     }
+
+    /**
+     * @test
+     */
+    public function can_add_categories()
+    {
+        $rating = Rating::factory()->create(['categories' => 0]);
+        $rating->addCategory(Rating::CREATIVE_BITMASK);
+        $this->assertTrue($rating->hasCategory(Rating::CREATIVE_BITMASK));
+    }
+
+    /**
+     * @test
+     */
+    public function has_category_helper_function_test()
+    {
+        $rating = Rating::factory()->create(['categories' => 0]);
+        $rating->categories = Rating::FRIENDLY_BITMASK;
+        $rating->save();
+        $this->assertTrue($rating->hasCategory(Rating::FRIENDLY_BITMASK));
+        $this->assertFalse($rating->hasCategory(Rating::CREATIVE_BITMASK));
+    }
+
+    /**
+     * @test
+     */
+    public function can_have_multiple_categories()
+    {
+        $categories = Rating::CREATIVE_BITMASK + Rating::FRIENDLY_BITMASK;
+        $rating = Rating::factory()->create(['categories' => $categories]);
+        $this->assertTrue($rating->hasCategory(Rating::CREATIVE_BITMASK));
+        $this->assertTrue($rating->hasCategory(Rating::FRIENDLY_BITMASK));
+    }
+
+    /**
+     * @test
+     */
+    public function can_remove_categories()
+    {
+        $categories = Rating::CREATIVE_BITMASK + Rating::FRIENDLY_BITMASK;
+        $rating = Rating::factory()->create(['categories' => $categories]);
+        $rating->removeCategory(Rating::FRIENDLY_BITMASK);
+        $this->assertTrue($rating->hasCategory(Rating::CREATIVE_BITMASK));
+        $this->assertFalse($rating->hasCategory(Rating::FRIENDLY_BITMASK));
+    }
+
+    /**
+     * @test
+     */
+    public function can_get_category_labels()
+    {
+        $categories = Rating::FLEXIBLE_BITMASK + Rating::FRIENDLY_BITMASK + Rating::CREATIVE_BITMASK + Rating::HELPFUL_BITMASK + Rating::PREPARED_BITMASK;
+        $rating = Rating::factory()->create(['categories' => $categories]);
+        $labels = $rating->getLabelsAttribute();
+
+        $this->assertContains(Rating::FLEXIBLE_LABEL, $labels);
+        $this->assertContains(Rating::HELPFUL_LABEL, $labels);
+        $this->assertContains(Rating::PREPARED_LABEL, $labels);
+        $this->assertContains(Rating::FRIENDLY_LABEL, $labels);
+        $this->assertContains(Rating::CREATIVE_LABEL, $labels);
+        $this->assertCount(5, $labels);
+    }
+
+    /**
+     * @test
+     */
+    public function can_scope_queries_to_categories()
+    {
+        $categories1 = Rating::FLEXIBLE_BITMASK + Rating::FRIENDLY_BITMASK;
+        $categories2 = Rating::HELPFUL_BITMASK + Rating::FRIENDLY_BITMASK;
+        $categories3 = Rating::PREPARED_BITMASK + Rating::HELPFUL_BITMASK;
+
+        $rating1 = Rating::factory()->create(['categories' => $categories1]);
+        $rating2 = Rating::factory()->create(['categories' => $categories2]);
+        $rating3 = Rating::factory()->create(['categories' => $categories3]);
```

Comment:
We don't really need to assign these to variable if they wont be used

```suggestion
        Rating::factory()->create(['categories' => $categories1]);
        Rating::factory()->create(['categories' => $categories2]);
        Rating::factory()->create(['categories' => $categories3]);
```